rest-service/manager_rest/resources_v2.py

```diff
@@ -21,12 +21,14 @@
 from datetime import datetime
 from collections import OrderedDict
 
-from flask_securest.rest_security import SecuredResource
 
+import elasticsearch
 from flask_restful_swagger import swagger
 from flask import request
 from flask.ext.restful import marshal
 
+from flask_securest.rest_security import SecuredResource
```

Comment:
That's a matter of taste. Although not entirely standard, for "3rd party" library imports that actually derive from Cloudify (e.g. commons imports rest client) I usually have a fourth section that comes between the actual 3rd party and the in package imports.

It is up to debate whether we treat flask_securest as external or internal